03week/airplane/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>UX Atlantic</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+
+  <body>
+    <header>
+
+      <!-- Make this div float the a tags -->
+      <div class="navbar">
+        <img src="./img/logo.png" class="logo" alt="logo">
+        <div class="navbarright">
+        <ul class="mainFeatures">
```

Comment:
for all of the following list items, it would be good to make both the image and the text a clickable link!